blueman/plugins/applet/NMDUNSupport.py

```diff
@@ -18,18 +17,20 @@ def __init__(self, parent, service, reply, err):
         self.rfcomm_dev = None
         self.timeout = None
 
-        self.parent.bus.add_signal_receiver(self.on_interfaces_added, "InterfacesAdded",
-                                            "org.freedesktop.DBus.ObjectManager")
+        self.signal_sub = self.parent._bus.signal_subscribe("org.freedesktop.ModemManager1",
+                                                            "org.freedesktop.DBus.ObjectManager",
+                                                            "InterfacesAdded",
+                                                            None,
+                                                            "/org/freedesktop/ModemManager1",
```

Comment:
I'm pretty sure `None` and `"/org/freedesktop/ModemManager1"` need to appear the other way around.